### PR TITLE
Pre-transfer hook for ArchivesSpace IDs

### PR DESCRIPTION
**Pull request by Hwesta**
Pre-transfer hook to generate ArchivesSpace IDs from the filename and writes to an archivesspaceids.csv

### CODE REVIEW

File: transfers/pre-transfer/archivesspace_ids.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    try:
+        os.mkdir(os.path.join(transfer_path, 'metadata'))
+    except OSError as e:
+        if e.errno == errno.EEXIST:
+            pass  # Already exists
+        else:
+            raise
+    with open(os.path.join(transfer_path, 'metadata', 'archivesspaceids.csv'), 'w') as f:
+        writer = csv.writer(f)
+        for row in as_ids:
```
Comment (mistydemeo) [inline transfers/pre-transfer/archivesspace_ids.py]: How about:

```python
writer.writerows(as_ids)
```

**Comment by mistydemeo** [comment]
:+1: 

Is the lack of a header row consistent with the other tools?

File: transfers/pre-transfer/archivesspace_ids.py (hunk 1 above)
Comment (Hwesta) [inline transfers/pre-transfer/archivesspace_ids.py]: Good catch. Fixed.

**Comment by hakamine** [comment]
Would it be ok to merge to master once #20 is merged to dev/archivesspace ?

**Comment by Hwesta** [comment]
That seems reasonable.

**Pull request merged**